applications/eosreddit.md

```diff
@@ -0,0 +1,155 @@
+- **Reason for Interest:** EOS VISION proposes to support the EOS community on Reddit, which has 97.7k members, is being one of the largest EOS communities, it is also the one with the most FUD of all EOS, since a couple of years, It has been a bit abandoned, and has been recovering thanks to the team of EOS SUPPORT and EOS BEES. We have seen how necessary and important this social network can be to attract both new users and new projects, so we propose to support Reddit.
+
+### Project Details
+
+- Our Mission
+Educate and inform the crypto currency world about the most recent events about EOS and its technological improvements, generate active use of Reddit encouraging the adoption of EOS, promoting the different contents of interest to users on Reddit. 
+- Projects Secondary Objectives:
+Weekly informative bulletins and important information releases.
+Important podcasts with the EOS community.
+General support for the different EOS projects.
+Informative courses for new users in EOS.
+Monthly report of the work done in EOS VISION.
+- Oriented to solve current problems
+Interact and correct misconceptions about EOS on social networks, especially on twitter and Reddit, which have been based on price, as well as respectfully encourage users who make negative comments dedicated to FUD to check the new stage of EOS.
+- Documentation of core components, protocols, architecture, etc. to be deployed
+We intend to use the reports, news, videos and other RELEVANT information found in the EOS COMMUNITY, and thus broadcast it on the social networks of Reddit and Facebook.
+- What your project is _not_ or will _not_ provide or implement
+We will support Reddit, but it will not be owned by us, EOS REDDIT and EOS Facebook is owned by the community and anyone can provide their services.
+  - This is a place for you to manage expectations and to clarify any limitations that might not be obvious
+  - 3 PILLARS VISIONS REDDIT.
+1.Make different daily Posts regarding the EOS Ecosystem: Maintaining activity within Reddit is one of the most important ways to eradicate FUD, demonstrating that it is an active community and that good opportunities, good projects are being created and there are new news for read, share and comment at all times.
+
+2. Increase in daily moderation: A moderation of all the SPAM that contains the official EOS reddit will be carried out, as well as the active moderation queue that may exist, it is worth mentioning that the SPAM queue has accumulated in recent months due to the number of bots that enter our Reddit.
+
+3. Reddit-Discord Reward System: Based on the “Tip.cc” Reward system in Discord, and the EOS BEES idea of ​​performing “SWARM PAID” to different important Twitter Posts, we will bring this idea to Reddit, dedicate 2 minutes a day to Vote on our Reddit, check out the “VISIONS REDDIT” channel where we will upload active posts daily, here you can vote and post in our thread that you have voted so that you can earn daily rewards.
```

Comment:
@Creamyvanill this `3.` section is out of scope of the current milestones, when first reading it seems like it was included in the initial grant request.

Perhaps could state that this section is in (Future Plan).

Reply:
Good point, I will update it, for a future plan, just as I will update that the ENF already has its account on Reddit. @DenisCarriere